openid-connect-4-credential-issuance-1_0.md

```diff
@@ -540,7 +684,7 @@ HTTP/1.1 200 OK
 }
 ```
 
-### Credential Response (deferred flow)
+### Deferred Credential Response
```

Comment:
I think there is another endpoint missing where the RP uses the acceptance token to fetch the credential.

Reply:
put a placeholder